ddtrace/contrib/django/patch.py

```diff
@@ -177,10 +177,14 @@ def traced_cache(django, pin, func, instance, args, kwargs):
                 db.ROWCOUNT, sum(1 for doc in result if doc) if result and isinstance(result, Iterable) else 0
             )
         elif command_name == "get":
+            # check if we used default value, and set rowcount to 0 if result is equal to the default
+            if result in args or ("default" in kwargs and kwargs["default"] == result):
```

Comment:
I realize this is the row count we might be using in other cache miss cases. In this case, where a default is provided, is there any other metadata on the span that would tell you that the cache get had a default?

Reply:
Not currently.